daemon/src/model/cfd.rs

```diff
@@ -310,6 +310,7 @@ pub enum CfdEvent {
     ContractSetupStarted,
```

Comment:
We compute the fee together with the `Started` event. Would it make more sense to store the fee in here instead?

Reply:
not really - because you would suddenly need to reason about all the failures/rejections, which make things heaps more complicated for no good reason. I opted to put the information in when we can be sure that the user *has been* charged.